src/mame/apple/apple3_m.cpp

```diff
@@ -1294,6 +1294,46 @@ void apple3_state::ay3600_data_ready_w(int state)
 	}
 }
 
+void apple3_state::ay3600_ako_w(int state)
+{
+	m_anykeydown = (state == ASSERT_LINE) ? true : false;
+
+	if (m_anykeydown)
+	{
+		m_repeatdelay = 5;
+		m_repttimer->adjust(attotime::from_hz(10));
+	}
```

Comment:
Wouldn’t it be simpler to just set the initial delay of half a second here rather than counting five ticks?  Also, do you want to cancel the timer when AKO is deasserted?

Reply:
Ahh, yes, that's a result of me copying it over and then adding the faster speed later and not thinking it over again. It's much cleaner coded like that.  I had thought no need to cancel it, but it makes it nice and clean so have added it. Thanks for your patience working through these updates with me.

Reply:
No problem.  Thanks for following up.